content/docs/1.5.4/monitoring/prometheus-and-grafana-setup.md

```diff
@@ -61,9 +63,26 @@ Follow instructions in [Prometheus Operator - Quickstart](https://github.com/pro
       - port: manager
     ```
 
-    Longhorn ServiceMonitor has a label selector `app: longhorn-manager` for selecting Longhorn backend service.
+#### Install Longhorn ServiceMonitor with Helm
+
+1. Modify the YAML file `longhorn/chart/values.yaml`.
+
+    ```yaml
+    metrics:
+      serviceMonitor:
+        # -- Setting that allows the creation of a Prometheus ServiceMonitor resource for Longhorn Manager components.
```

Comment:
```suggestion
        # -- Setting that allows the creation of a [Prometheus Operator](https://prometheus-operator.dev/) ServiceMonitor resource for Longhorn Manager components.
```

Reply:
Modified with 1.5.4, 1.6.0/1, and 1.7.0. Thanks.